core/src/engine/task/task_processor_config.hpp

```diff
@@ -18,6 +18,8 @@ enum class OsScheduling {
   kIdle,
 };
 
+enum class TaskProcessorQueue { kMoodyCamelTaskQueue, kWorkStealingTaskQueue };
```

Comment:
Implement `Parse` for this enum. See parsing for `OsScheduling` below as an example